wvm-apps/wvm-exexed/crates/reth-exexed/src/exex/ar_actor.rs

```diff
@@ -1,16 +1,12 @@
-use crate::{network_tag::get_network_tag, new_etl_exex_biguery_client};
+use crate::network_tag::get_network_tag;
```

Comment:
we should bump versions in `wvm-apps/wvm-exexed/crates/reth-exexed/src/constant.rs`